charts/enterprise-metrics/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 1.7.2
+version: 1.7.3
```

Comment:
I'm not sure if bumping this to 1.7.3 is the right thing to do, please take a look.